torchaudio/prototype/ctc_decoder/__init__.py

```diff
@@ -0,0 +1,15 @@
+import torchaudio
+
+try:
+    torchaudio._extension._load_lib('libtorchaudio_decoder')
+    from .ctc_decoder import KenLMLexiconDecoder, kenlm_lexicon_decoder
+except ImportError:
+    raise ImportError(
+        "flashlight decoder bindings are required to use this functionality. "
+        "Please set BUILD_CTC_DECODER=1 when building from source."
+    )
```

Comment:
Since when `libtorchaudio_decoder` is available, `_torchaudio_decoder` is also available, it is sufficient to wrap `torchaudio._extension._load_lib('libtorchaudio_decoder')` with try except. `from .ctc_decoder import KenLMLexiconDecoder, kenlm_lexicon_decoder` can stay outside of the `try ~ except ~`. (In general, the smaller the statements wrapped with `try ~ except`, the better)

Reply:
[`_load_lib`](https://github.com/pytorch/audio/blob/main/torchaudio/_extension.py#L23) does not actually throw an error if the library does not exist (it will just load nothing) -- should we change this behavior to throw an error or warning if called with a nonexistent library?

Reply:
> [`_load_lib`](https://github.com/pytorch/audio/blob/main/torchaudio/_extension.py#L23) does not actually throw an error if the library does not exist (it will just load nothing) -- should we change this behavior to throw an error or warning if called with a nonexistent library?

Oops. You are right. This one is tricky, we want to allow the skipping behavior as well for the sake of PEX-format compatibility, which deploys `libtorchaudio*.so` separately. We want to have the same mechanism for prototype CTC decoder.

Okay, so it's more appropriate to wrap the both imports. Thanks for the clarification.